### PR TITLE
Food.pm merge for the nutrients

### PR DESCRIPTION
**Pull request by teolemon**
More nutrient merges

### CODE REVIEW

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -2828,220 +2982,313 @@ fluoride => {
...
-	es => "Huella de carbono / Emisiones de CO2",
-	el => "Αποτύπωμα άνθρακα/Εκπομπές CO2",
-	it=> "Emissioni di CO2 (impronta climatica)",
-	pt => "Pegada de carbono / Emissões de CO<sub>2</sub>",
-	de => "Carbon Footprint / CO2-Emissionen",
+	en => "Carbon footprint / CO<sub>2</sub> emissions",
+	cs => "Uhlíková stopa / Emise CO<sub>2</sub>",
+	de => "Carbon Footprint / CO<sub>2</sub>-Emissionen",
+	el => "Αποτύπωμα άνθρακα/Εκπομπές CO<sub>2</sub>",
+	es => "Huella de carbono / Emisiones de CO<sub>2</sub>",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm]: <pre>Illegal declaration of anonymous subroutine at /home/travis/build/openfoodfacts/openfoodfacts-server/lib/ProductOpener/Food.pm line 3272.

Comment (stephanegigandet) [reply lib/ProductOpener/Food.pm]: [Sat Nov 10 17:09:39 2018] Food.pm: Bareword found where operator expected at /home/travis/build/openfoodfacts/openfoodfacts-server/lib/ProductOpener/Food.pm line 3267, near ""carbon"
[Sat Nov 10 17:09:39 2018] Food.pm:   (Might be a runaway multi-line "" string starting on line 3264)

File: lib/ProductOpener/Food.pm [hunk 2]
```diff
@@ -2231,118 +2278,139 @@ salt => {
...
-	et => "Niatsiin",
 	pl => "Niacyna",
-	lt => "Niacinas",
-	mt => "Niaċina",
-	sk => "Niacín",
+	pt => "Vitamina B3 / Vitamina PP (Niacina)",
 	ro => "Niacină",
-	bg => "Ниацин",
-	zh => "维生素B3(烟酸)",
+	ru => "Ниацин / Витамин PP / Витамин B<sub>3</sub>>",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm]:
```suggestion
	ru => "Ниацин / Витамин PP / Витамин B3",
```

File: lib/ProductOpener/Food.pm [hunk 3]
```diff
@@ -3053,13 +3300,21 @@ ph => {
...
 	fr => "Fruits, légumes et noix (minimum)",
 	es => "Frutas, verduras y nueces (mínimo)",
 	el => "Φρούτα, λαχανικά, καρποί (ελάχιστο)",
+	hu => "Gyümölcsök, zöldségek és diófélék (minimum)",
 	nl => "Fruit, groenten en noten (minimum)",
 	nl_be => "Fruit, groenten en noten (minimum)",
-	de => "Obst, Gemüse und Nüsse (Minimum)",
-	unit => "%",
+	pt => "Frutas, legumes e nozes (mínimo)",
+	zh_TW => "水果、蔬菜、堅果（最低限度）",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm]:
```suggestion
	zh_TW => "水果,蔬菜,堅果（最低限度）",
```

File: lib/ProductOpener/Food.pm [hunk 4]
```diff
@@ -3053,13 +3300,21 @@ ph => {
...
+	zh_HK => "水果、蔬菜、堅果（最低限度）",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm]:
```suggestion
	zh_HK => "水果,蔬菜,堅果（最低限度）",
```

File: lib/ProductOpener/Food.pm [hunk 5]
```diff
@@ -3053,13 +3300,21 @@ ph => {
...
+	zh_CN => "水果、蔬菜、坚果（最低限度）",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm]:
```suggestion
	zh_CN => "水果,蔬菜,坚果（最低限度）",
```

File: lib/ProductOpener/Food.pm [hunk 6]
```diff
@@ -3053,13 +3300,21 @@ ph => {
...
+	zh => "水果、蔬菜、坚果（最低限度）",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm]:
```suggestion
	zh => "水果,蔬菜,坚果（最低限度）",
```

**Comment by hangy** [comment]
@teolemon this PR has conflicts. Is it something you still want to merge?

**Comment by teolemon** [comment]
The nutrients have just been taxonomized.

**Pull request closed without merging**